install/kots/manifests/gitpod-installation-status.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: installation-status
+  labels:
+    app: gitpod
+    component: gitpod-installer-status
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gitpod
+      component: gitpod-installer-status
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: gitpod
+        component: gitpod-installer-status
+    spec:
+      restartPolicy: Always
+      serviceAccountName: installer
+      containers:
+        - name: installation-status
+          # This will normally be the release tag
+          image: "eu.gcr.io/gitpod-core-dev/build/installer:main.2968"
+          command:
+            - /bin/sh
+            - -c
+          args:
+            - |
+              set -e
+
+              while true
+              do
+                echo "Checking installation status"
+
+                if [ "$(helm status -n {{repl Namespace }} gitpod -o json | jq '.info.status == "deployed"')" != "true" ];
+                then
+                  echo "Gitpod: Installation not complete"
+                  exit 1
```

Comment:
Question: (I haven't tested this yet) when an upgrade is happening wouldn't a new deployment be created of the `installation-status` deployment too? wouldn't KOTS UI show its status as `Ready` referring from the old deployment?

Reply:
Theoretically yes, but only for 10 seconds whilst the sleep waits for the next try. I think that's an acceptable compromise.

In practice, I've not found that to be the case though